draggableMotion/src/main/ets/components/MainPage/types.ets

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
Refactor the filename to DraggableTypes.ets

Reply:
done